frontend/src/concepts/k8s/useStorageClasses.ts

```diff
@@ -0,0 +1,6 @@
+import useFetchState from '~/utilities/useFetchState';
+import { getStorageClasses } from '~/api';
+
+const useStorageClasses = () => useFetchState(getStorageClasses, []);
+
```

Comment:
@andrewballantyne this check seems new, was not there in incubation. Do you want me or Alex to fix this directly in f/storage-class?

Reply:
@alexcreasy nice resolve of the error, thank you. In line with how it's done in most other places in the code.

Reply:
yup, good and in line with how it's done elsewhere. Thank you.